src/pages-helpers/curriculum/docx/builder/1_frontCover.ts

```diff
@@ -44,10 +46,20 @@ export default async function generate(
 
   const phaseTitle = keyStageFromPhaseTitle(data.phaseTitle);
   const subjectTitle = data.subjectTitle;
-  const examboardTitle = data.examboardTitle
-    ? `${data.examboardTitle} (KS4)`
+
+  const examboardTitle = data.examboardTitle ? `${data.examboardTitle}` : "";
+  const tierTitle = slugs.tierSlug ? `${capitalize(slugs.tierSlug)}` : "";
```

Comment:
Is there not a `*Title` we can use for these?

Reply:
I think `mv_lesson_listing_5_0_1` gets `tierTitle` as a reference.

Reply:
Creating a tech debt ticket for this here for future child subjects: https://www.notion.so/oaknationalacademy/DOCX-Child-subject-tier-filtering-title-on-title-page-should-come-from-MV-data-5f340bc9b3024de38be4adc5b770c938?pvs=4